security/access_token.rst

```diff
@@ -343,6 +343,438 @@ and configure the service ID as the ``success_handler``:
     ``failure_handler`` option and create a class that implements
     :class:`Symfony\\Component\\Security\\Http\\Authentication\\AuthenticationFailureHandlerInterface`.
 
+Using OIDC
+----------
+
```

Comment:
It would be nice to have a short paragraph describing what OIDC is, eventually mentioning a few identity/access providers such as Keycloack, especially for people that not yet familiar with the protocol.